src/components/auth/AuthWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from "react";
+
+import { MsalProvider } from "@azure/msal-react";
+import { Configuration, PublicClientApplication } from "@azure/msal-browser";
+
+const configuration: Configuration = {
+  auth: {
+    clientId: "276fd164-c30c-485f-bd76-a8f3b5bcac74",
```

Comment:
`clientId` is not a secret and so is OK to keep in VCS (I believe).  See [OAuth specification](https://tools.ietf.org/html/rfc6749#section-2.2) and [StackOverflow](https://stackoverflow.com/questions/61613985/are-azure-application-client-ids-safe-to-store-in-the-frontend).

Deliberately chose to keep the auth configuration with the `AuthWrapper` component (i.e. not separate into `authConfig.json` or similar) so there is one place for all things 'auth'.